app/helpers/application_helper/button/vm_html5_console.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationHelper::Button::VmHtml5Console < ApplicationHelper::Button::Bas
   needs :@record
 
   def visible?
-    %w[vnc webmks spice].any? { |type| @record.send(:console_supported?, type) }
+    @record.supports?(:html5_console)
```

Comment:
Wow is this really it?

Reply:
I think we can probably rewrite https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/button/vm_vmrc_console.rb and https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/button/vm_native_console.rb to be more generic (aka drop the vendor == vmware stuff)

Reply:
Ok, I read change them to just use "supports?"

Do we have an opinion on whether to change the supports to add some of the other "this vm is on" kind of logic that we are putting in the validates?

Are we thinking we should drop that validates method all together? (I'm suggesting it)